src/kirby/components/list/list-cell/list-cell.component.html

```diff
@@ -0,0 +1,11 @@
+<!--
+    TODO: loose the div and use the host element
+   don't use ngStyle - maybe scss instead??? 
+   Same width for all cols
+
+-->
+<div class="cell" [ngStyle]="{'justify-content': alignVertical, 'align-items' : alignHorisontal}">
```

Comment:
Looks like your editor is set to have tabs 4 spaces long and not 2 like the rest of the solution?